service/history/service.go

```diff
@@ -114,8 +114,8 @@ func NewConfig(dc *dynamicconfig.Collection, numberOfShards int) *Config {
 		TimerProcessorGetFailureRetryCount:                 5,
 		TimerProcessorCompleteTimerFailureRetryCount:       10,
 		TimerProcessorUpdateShardTaskCount:                 100,
-		TimerProcessorUpdateAckInterval:                    1 * time.Minute,
-		TimerProcessorCompleteTimerInterval:                1 * time.Second,
+		TimerProcessorUpdateAckInterval:                    5 * time.Second,
```

Comment:
Updating shard is a very expensive query.  I'm not sure changing update interval for all 16K shards is a good idea.

Reply:
we have the shard level min update interval to guarantee shard is not updated frequently (on the database level), however, on the application level (in memory), as frequent as possible